### ISSUE TITLE
Not possible to specify chapter when multiple with same name exist?

### ISSUE THREAD

**Issue by ScherisMarie** [frozenpandaman/mangadex-dl#31]
https://mangadex.org/title/51497/the-villainess-became-a-commoner

When I ran this script last night, the chapter list returned “1, 1, 2” for the results to choose to download.

Unless I’m missing something, typing in “1” only downloaded the first iteration of Ch1, so I had to download them all to get the second one that I actually wanted.

Did I miss something or is there no way to currently specify which one to download when multiple of the same chapter number exist?

**Comment by frozenpandaman** [comment]
Not currently possible, sorry! From the readme:
> **Current limitations**  
> * The script will download all available releases (in your language) of each chapter specified.

At some point I'll figure this out but hopefully for now it's not too limiting.

**Comment by frozenpandaman** [comment]
@ScherisMarie What would your preferred UI/UX be for this? e.g. if you type in "1", should all the groups that have released a chapter numbered 1 be displayed, and there's a prompt asking you to specify which to download every time this occurs?

**Comment by ScherisMarie** [comment]
That would work. Assuming the "1" listings in that example would show the group names to tell them apart?

**Comment by frozenpandaman** [comment]
Yep, that's what I meant by "all the groups that have released a chapter numbered 1 [would] be displayed" haha. Thanks!

**Comment by orbea** [comment]
A prompt for each individual chapter on a long series with more than one translation group might be somewhat tedious. :)

**Comment by ArjixWasTaken** [comment]
I suggest for the user to save in a json file a priority list, where they will have the group ids they want to be prioritized.
That way when there are multiple groups, it will look at the json, if the group is in that json and its ranked higher than the other groups, it shall download that.
And if that group is not included in the priority list and no group from that list has a release of that chapter then it picks a random one.

I can try adding this as a PR.
